gh-codeql

```diff
@@ -220,15 +220,24 @@ function install_stub() {
     fi
 
     if [ -w "$bindir" ]; then
+        if ["$platform" == "win64"]; then
+          cat << "_codeql_stub" > "$bindir/codeql.cmd"
+@echo off
 
-        cat << "_codeql_stub" > "$bindir/codeql"
+gh codeql %*
```

Comment:
we should add an `exit /b %ERRORLEVEL%` line, otherwise a command failure gets lost